postgres/datadog_checks/postgres/metadata.py

```diff
@@ -0,0 +1,122 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from typing import Dict, Optional, Tuple  # noqa: F401
+
+import time
+import json
+import psycopg2
+
+try:
+    import datadog_agent
+except ImportError:
+    from ..stubs import datadog_agent
+
+from datadog_checks.base import is_affirmative
+from datadog_checks.base.utils.db.utils import (
+    DBMAsyncJob,
+    RateLimitingTTLCache,
+    default_json_event_encoding,
+    obfuscate_sql_with_metadata,
+)
+from datadog_checks.postgres.connections import MultiDatabaseConnectionPool
+from datadog_checks.base.utils.tracking import tracked_method
+
+# default collection interval in minutes
+DEFAULT_COLLECTION_INTERVAL = 30
+
+PG_SETTINGS_QUERY = """
+SELECT name, setting, category, short_desc FROM pg_settings
```

Comment:
I think this stuff is stored  in the system catalogs, which requires a disk read but then it should (in theory) be served from memory? 

Seems OK to run this query once every 15s. I only had the interval so long because these settings aren't changed very often

Reply:
Kk let's just review and explain plan and latency before merge, but otherwise sounds good to me.